src/config.rs

```diff
@@ -36,13 +36,23 @@ impl Default for WatchConfig {
 
 #[derive(Debug, Serialize, Deserialize, PartialEq)]
 pub struct Config {
+    // When commit_exclude_git_config is false,
+    // never use any git configuration to sign dura's commits.
+    // Defaults to false
+    #[serde(default)]
+    pub commit_exclude_git_config: bool,
+    pub commit_author: Option<String>,
+    pub commit_email: Option<String>,
     pub repos: BTreeMap<String, Rc<WatchConfig>>,
 }
 
 impl Config {
     pub fn empty() -> Self {
         Self {
             repos: BTreeMap::new(),
+            commit_exclude_git_config: true,
```

Comment:
The comment above says defaults to `false` but it's being created as `true` here

Reply:
Good catch!